python/ray/tests/test_actor.py

```diff
@@ -974,5 +974,30 @@ def get_ref(self):
     assert ray.get(inner.ping.remote()) == "pong"
 
 
+def test_actor_autocomplete(ray_start_regular_shared):
```

Comment:
Looks fantastic! Does this test both the `ray.init` and `ray.client` code paths?

Reply:
Yep! This `test_actor.py` is also run against client mode ([see here](https://github.com/ray-project/ray/blob/45d2331d5a6062e894725024e2a608830ac0b7ce/python/ray/tests/BUILD#L231))!